org.opent2t.sample.multisensor.superpopular/test/multisensorTests.js

```diff
@@ -235,15 +235,19 @@ function runMultisensorTests(settings) {
             return OpenT2T.invokeMethodAsync(translator, SchemaName, 'getDevicesLastchanged', [deviceIds['opent2t.d.sensor.motion']])
                 .then((response) => {
                     t.is(response.rt[0], 'opent2t.r.timestamp');
-                    let testChangedAt = testSettings.inputLastReading["motion_changed_at"];
-                    let testUpdatedAt = testSettings.inputLastReading["motion_updated_at"];
-                    if ((!testChangedAt || isNaN(testChangedAt))
-                        && (!testUpdatedAt || isNaN(testUpdatedAt))) { 
-                        t.true(response.timestamp === undefined, "Expected undefined value for motion-lastchanged");
-                    }
-                    else {
+                    if(testSettings.inputLastReading !== undefined) { //Wink-Only
```

Comment:
Not blocking this PR, created issue [https://github.com/openT2T/translators/issues/223](https://github.com/openT2T/translators/issues/223)

---
In reply to: [109253159](https://github.com/openT2T/translators/pull/220#discussion_r109253159) [](ancestors = 109253159)